### PR TITLE
fixed the dialog on the menu button and adjusted other borders

### PR DESCRIPTION
**Pull request by connorvang**
## Description

Fixed the menu for the menu button being the wrong color (my bad) and adjusted other borders I I missed earlier.

## Todo

- [x] Bump version and add tag

### CODE REVIEW

File: src/components/simple-table/SimpleTable.module.scss [hunk 1]
```diff
@@ -4,10 +4,10 @@
   width: 100%;
 
   > thead {
-    text-transform: uppercase;
+    text-transform: none;
```
Comment (evad1n) [inline src/components/simple-table/SimpleTable.module.scss:7]: What's the reason for this change? In all the designs I have seen the headers are always capitalized.

Comment (connorvang) [reply src/components/simple-table/SimpleTable.module.scss:7]: We are slowly moving away from all caps across the board and this was one of the last remaining places in Amino where that text-transform is applied.

**Review by evad1n — APPROVED** [review]

**Review by evad1n — APPROVED** [review]
